extensions/amp-list/amp-list.md

```diff
@@ -127,7 +127,13 @@ In `<amp-list>`, you can use the [`items`](#items-optional) attribute to render
 
 ### Specifying an overflow
 
-Optionally, the `<amp-list>` element can contain an element with an `overflow` attribute. This element is shown if the AMP Runtime cannot resize the `<amp-list>` element as requested, which is the case if the list is inside the viewport. If the list is outside the viewport, it will be automatically expanded.
+Optionally, the `<amp-list>` element can contain an element with the `overflow` attribute. The AMP `overflow` attribute to resize AMP components that fetch content if the AMP framework is unable to size the component as requested. An element inside of the AMP component with the `overflow` attribute will appear if one of the following conditions are met:
```

Comment:
"The AMP `overflow` attribute to resize AMP components that fetch content if the AMP framework is unable to size the component as requested."?

Reply:
"An element inside of the AMP component with the `overflow` attribute will appear if one of the following conditions are met"

It's not clear that the `overflow` element is on the child, not the amp component itself. Also, I think you're missing an "unless"

Reply:
I totally misread Cathy's comment - good catches all around, thanks!